### ISSUE TITLE
Consistency issue: Site title not present în meta

### ISSUE THREAD

**Issue by avoinea** [collective/collective.behavior.seo#6]
Site title should be automatically joined to seo_title here:

https://github.com/avoinea/collective.behavior.seo/blob/902021ebe79ba32d26bba1b142216ab4320b4e2c/src/collective/behavior/seo/browser/title.py#L18-L19

See initial code:

https://github.com/plone/plone.app.layout/blob/1c4bb6a5c1383a120d29c2fe9f9b3dec553c7bff/plone/app/layout/viewlets/common.py#L154

**Comment by fredvd** [comment]
@avoinea If I remember correctly, this is a design decision.  seo_title is an optional field. If you don't fill in anything then site title + page title is used, which is the default Plone behavior.  

seo_title allows to override that default 'merged' title. If seo_title would also merge site_title again, you cannot really customise it so it defeats the purpose of this addon.

**Comment by avoinea** [comment]
@fredvd I had a feeling about being a design decision, that's why I didn't open a PR right away :smile: 

Maybe we can add a checkbox or something like: `Append site title`

Otherwise is very easy to end-up with inconsistent meta titles.

**Comment by fredvd** [comment]
> Maybe we can add a checkbox or something like: `Append site title`

Yes that would be possible option/solution.  Then I'd call it  'still append Site title' and add in the Description (first attempt to formulate something):

'By default the site title is appended to the Page title for the SEO/metadata title. If you override the SEO Title in the above field this automatic behavior is disabled, unless you active this option.

---
But then we have to answer the next question: should this check box be enabled by default? For existing users of collective.behavior.seo it shouldn't, changes behavior of the add'on. And overriding the title is already an exception.  But if you expect that a lot of editors will creat inconsistent seo titles in a certain site and/or for new projects you'd want it to be enabled by default. 

We can store the default in as a registry / control panel option and try to put that as the default value so you can control the default behavior per site/instance. 

I remember having a fight with zope.schema in the past where the defaultFactory would override an explicitly set False on a boolean field when you'd request the field value. Difference between None, NULL / False). But that could be my not understanding/using those fields correctly.

**Comment by avoinea** [comment]
:thinking: True, the setting should at site/registry level, otherwise we just add more complexity and we don't fix the issue as the editors will still be able to create inconsistent titles.
